### PR TITLE
adds ability to add description to swagger security scheme object

### LINKED ISSUE #655: swagger Security Scheme Object description field
a developer should be able to add a description to the security scheme object, per the swagger specification:

screen shot 2017-04-25 at 11 53 22 am

**I will work on this (and feel free to add me on other swagger/scalatra issues)**

### PR DESCRIPTION
**Pull request by jb08**
resolves #655 

with description:
with description

with no description:
with no description

### CODE REVIEW

File: swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala [hunk 1]
```diff
@@ -420,7 +421,7 @@ class SwaggerSpec2 extends ScalatraSpec with JsonMatchers {
     val JObject(definitions) = j
     definitions.map {
       case (name, definition) =>
-        verifyFields(definition, swaggerJsonJValue \ "securityDefinitions" \ name, "type", "name", "in", "flow", "authorizationUrl", "scopes")
+        verifyFields(definition, swaggerJsonJValue \ "securityDefinitions" \ name, "type", "name", "description", "in", "flow", "authorizationUrl", "scopes")
```
Comment (takezoe) [inline swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala]: Perhaps you should add "description" field to `verifyAuthorizations()` as well.
https://github.com/scalatra/scalatra/blob/3af64ba1c2011db5f760253958b5c3bf0c76424e/swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala#L129

Comment (jb08) [inline swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala]: luckily the `description` field is not part of the [swagger 1.2 spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/1.2.md), so no need to implement or test. It is tested in `SwaggerSpec2`. Do you agree?

Comment (takezoe) [inline swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala]: Oh really! So you don't need to add "description" field in `SwaggerSerializers.scala`, right?

Comment (jb08) [inline swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala]: you're right!! I've re-made that change in the most recent commit.

**Comment by takezoe** [comment]
LGTM 👍

**Pull request merged**